test/list_notifier_test.dart

```diff
@@ -5,8 +5,8 @@ import '../lib/src/list_notifier.dart';
 
 void main() {
   group("Tests for the ListNotifier's methods", () {
-    late ListNotifier list;
-    List result = [];
+    late ListNotifier<int> list;
```

Comment:
yep, good to use types here

Reply:
As the new default it to notify always we should set mode to normal for the tests.

Reply:
@escamoteur, what do you mean by "address this comment"? 🤔👀

Reply:
I meant this comment:
>As the new default it to notify always we should set mode to normal for the tests.

if you change that you will have to review the tests again